My commit- Veronica

braduveronik opened the pull request.

braduveronik committed:
My commit- Veronica

enszus commented:
You have some conflicts that need to be fixed before being able to merge your pull request.

braduveronik committed:
myHomework_0

# Conflicts:
#	TabedApplication/Base.lproj/Main.storyboard
#	TabedApplication/FirstViewController.swift

enszus reviewed (approved):
Everything looks good, approved.
